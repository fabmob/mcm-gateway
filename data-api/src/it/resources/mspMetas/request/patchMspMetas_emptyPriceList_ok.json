{
  "hasStation": true,
  "logoUrl": "patchedLogoURL",
  "priceList":{
    "duration": [
      {
        "lowerPriceLimit": 12211,
        "upperPriceLimit": 22,
        "fixedFare": 3,
        "farePerUnit": 4,
        "unit": 5
      },
      {
        "lowerPriceLimit": 123,
        "upperPriceLimit": 2,
        "fixedFare": 3,
        "farePerUnit": 4,
        "unit": 5
      }
    ]
  }
}