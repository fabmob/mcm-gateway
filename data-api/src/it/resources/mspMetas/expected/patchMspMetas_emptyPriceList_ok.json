{
  "hasParking": false,
  "hasStationStatus": true,
  "primaryColor": "#0054BB",
  "type": "TROTTINETTE",
  "operator": "Dott",
  "url": "https:\/\/www.cityscoot.eu\/paris\/",
  "logoUrl": "patchedLogoURL",
  "priceList":{
    "duration": [
      {
        "unit": 5,
        "fixedFare": 3,
        "farePerUnit": 4,
        "priceListItemId": "f5e6cc23-740c-44f1-87bb-96762b35490b",
        "lowerPriceLimit": 123,
        "upperPriceLimit": 2
      },
      {
        "unit": 5,
        "fixedFare": 3,
        "farePerUnit": 4,
        "priceListItemId": "f5e6cc23-740c-44f1-87bb-96762b35490a",
        "lowerPriceLimit": 12211,
        "upperPriceLimit": 22
      }
    ],
    "parkingForbiddenFee": null,
    "distance": [],
    "priceListId": "f5e6cc23-740c-44f1-87bb-96762b35490c",
    "outOfBoundFee": null,
    "comment": null
  },
  "mspId": "f5e6cc23-740c-44f1-87bb-96762b35490d",
  "hasVehicle": null,
  "hasStation": true,
  "hasPrefParkingZone": false,
  "urlWebview": false,
  "isEnabled": null,
  "name": "Dott",
  "hasSpeedLimitZone": false,
  "logoFormat": "PNG",
  "hasNoParkingZone": false,
  "hasHold": false,
  "hasOperatingZone": false,
  "secondaryColor": "#0054BB"
}